ajax/libs/floating.js/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "floating.js",
+  "description": "Float a number of things up on a page (hearts, flowers, ...)",
+  "homepage": "https://haroen.me/floating.js/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Haroenv/floating.js.git"
+  },
+  "author": "Haroen Viaene",
+  "license": "Apache-2.0",
+  "filename": "floating.js",
+  "keywords": [
```

Comment:
@dakshshah96 can you please update keywords like https://github.com/Haroenv/floating.js/blob/c10660a50ee0679a0aaed0867376529a23af9fca/package.json#L18-L23